sys-cluster/charliecloud/charliecloud-0.34.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_COMPAT=( python3_{9..12} )
```

Comment:
```suggestion
PYTHON_COMPAT=( python3_{10..12} )
```

Reply:
Thanks, applied! 
Is there a rule on when to drop old langs?

Reply:
You can drop old targets when they are removed. For example we have removed python3.9 support globally (you can see this is now a NOP), so you can drop it. Another way to see it is by running `pkgcheck scan --commits`, which would flag all packages in the current patch set that retain the old COMPAT value.

Reply:
Thanks, that's very helpful! :+1: